infraohjelmointi_api/migrations/0046_coordinatornote.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 4.1.3 on 2023-09-28 07:21
+
+from django.conf import settings
+from django.db import migrations, models
+import django.db.models.deletion
+import uuid
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('infraohjelmointi_api', '0045_auto_project_finances_to_frame_finances'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='CoordinatorNote',
+            fields=[
+                ('id', models.UUIDField(default=uuid.uuid4, editable=False, primary_key=True, serialize=False)),
+                ('coordinatorNote', models.TextField(blank=True, default='')),
+                ('year', models.CharField(max_length=4)),
+                ('planningClass', models.CharField(max_length=150)),
+                ('updatedByFirstName', models.CharField(max_length=50)),
```

Comment:
updatedByFirstName and updatedByLastName can be removed from the model since the required information can be fetched from the updatedById foreign key relation.

CoordinatorNoteSerializer can have the field `updatedById = WhoAmISerializer(read_only=True)` to have the user object in the output as well.

Check ProjectNoteGetSerializer.py  line 10 for example